tests/e2e/tests/bookinfo/demo_test.go

```diff
@@ -412,7 +412,7 @@ func TestDbRouting(t *testing.T) {
 
 	respExpr := "glyphicon-star" // not great test for v2 or v3 being alive
 
-	_, err = checkHTTPResponse(u1, tc.gateway, respExpr, 11)
+	_, err = checkHTTPResponse(u1, tc.gateway, respExpr, 10)
```

Comment:
we really need another means of checking this. I suggest killing this test altogether. we have a test for metrics to ratings-v2 in mixer_test.go now. we can augment that to ensure we are using ratings-v2 and kill this test.  thoughts?

Reply:
I don't mind killing it. @ldemailly ??

Reply:
this test in theory tests the addition of the db (switch from not using to using it) but I suppose we can have it "always on"
I actually would like to have a test to ensure we don't loose mysql either (but it's a bit hard to test that from the product page)

I missed the v2 test in mixer, are you sure ? also this one is a different setup, it doesn't hurt to have 2 tests for the feature

let's keep it and I'll extend it to updating the db maybe or using lower / different number of stars

Reply:
I guess the images must not match the source if this 11->10 is changed but not the values
no big deal I guess we'll fix it later